static/app/types/organization.tsx

```diff
@@ -106,8 +108,16 @@ export type Member = {
   projects: string[];
   role: string;
   roleName: string;
-  roles: MemberRole[]; // TODO(ts): This is not present from API call
-  teams: string[];
+  roles: OrgRole[]; // Deprecated: use orgRoleList
```

Comment:
Is the key `role` deprecated here? or the type `OrgRole` deprecated here?

Why not just remove it here?

Reply:
`roles` is being deprecated, but I'm keeping it around for backwards compatibility as this key has been around for years. There's no desire to remove the key until we have some sort of API versioning.

I think it's worth explicitly documenting that it's deprecated, rather than have the FE ignore its existence from the BE response.

There's an error on `OrgRole`/`TeamRole` because it's dependent on another PR to add it in. I'll rebase when that PR is merged.